lib/spider/configs/eval.yaml

```diff
@@ -7,7 +7,7 @@ defaults:
   - prompts: default
 
 eval_type: "all"
-max_samples: null
+max_samples: 3
```

Comment:
The main configs should use the most sensible defaults, so let's not put a `max_samples` value (it's better to start a full training & realize you only want a few samples & stop the training than start a training thinking it's a full one & realize the next day it ran only on 3 samples)